bin-dir/system-test-script.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+cd ..
+
+echo "starting collection roller"
+bin/collection-roller &> /dev/null &
+
+echo "starting alert engine"
+bin/alert-engine &> /dev/null &
+
+echo "starting log collector"
+bin/log-collector &> /dev/null &
+
+echo "starting spark application"
+cd /home/username/pulse/log-example/
+./spark-logging &> /dev/null &
+sleep 10 &
```

Comment:
Why sleep here instead of waiting for the process to finish?

Reply:
Spark job is running for longer period and not stopping when we added wait. We are not able to move to curl command to assert records.